src/status_im2/contexts/chat/messages/content/text/view.cljs

```diff
@@ -75,15 +75,17 @@
 
 
 (defn render-block
-  [blocks {:keys [type literal children]} chat-id]
+  [blocks {:keys [type literal children]} chat-id style-override]
```

Comment:
see my comment above 👍

Reply:
It was just a notation by Icaro, Nothing to be done here.